docs/pages/database-access/guides/aws-dynamodb.mdx

```diff
@@ -258,3 +257,19 @@ $ aws dynamodb list-tables --endpoint-url=http://localhost:8000
     ]
 }
 ```
+
+You can also connect to this database from the AWS NoSQL Workbench, as documented in our [Database Access GUI Clients](../../connect-your-client/gui-clients.mdx#nosql-workbench) guide.
+
+You can also use this tunnel for programmatic access. The example below uses the `boto3` SDK from AWS:
+
+```python
```

Comment:
```suggestion
```code
```
or `shell`. Not sure what the docs preference is here, but the snippet example is a shell session

Reply:
You are technically correct (the best kind of correct), but using `code` results in no syntax highlighting for the python code:

![image](https://user-images.githubusercontent.com/2349184/211604325-b7d3e4c5-fb91-4554-9aea-c65f88ca9dc8.png)

Compared to using `python`:

![image](https://user-images.githubusercontent.com/2349184/211604435-488ebc80-cd9a-475a-8776-86889b653f73.png)

IMO, the highlighting is more useful than having an extra copy button that literally copies `python3` for the reader.